docs/adrs/notifications.md

```diff
@@ -0,0 +1,18 @@
+# Notifications
+
+## Context
+
+superwerker creates an OpsItem in Systems Manager OpsCenter for emails received by the RootMail feature. Without notifications for new OpsItems, users need to manually check the OpsCenter for new items.
+
+## Decision
```

Comment:
We could document why we did not use SES?

Reply:
> We decided against using SES because SNS subscriptions and email verification work out-of-the-box in CloudFormation